Cargo.toml

```diff
@@ -24,8 +24,8 @@ futures = "0.3.30"
 hex = "0.4.2"
 http-types = "2.6.0"
```

Comment:
cargo version bump?

Reply:
I was thinking to bump it a new PR to bump cargo/py version and also update changelog

Reply:
what do you think?